main.js

```diff
@@ -0,0 +1,70 @@
+
+     
+
+var snakewaterMontana = {
+    paleontologist: "Dr. Alan Grant",
+    depth: "10 meters",
+    specimen: "Velociraptor"
+  }
+  var guestOfHonor = snakewaterMontana.paleontologist;
+
+     var cleverGirl = snakewaterMontana.specimen;
+
+     console.log (guestOfHonor);
+ console.log (cleverGirl);
+
+ nicaragua = {
+    depth: "200 meters",
+    annualBudget: 1500000,
+    specimens: [
+      "Tyrannosaurus Rex",
+      "Stegosaurous",
+      "Triceratops",
+      "Velociraptor"
+    ]
+  }
+ 
+  annualBudget = annualbudget + 250000;
+  console.log (annualBudget);
+
+  var favoriteSpecimen = specimens[2]
```

Comment:
To access the specimens array inside an object you need to call the object first ` var favoriteSpecimen =  nicaragua.specimens[2]`